lib/elasticity/version.rb

```diff
@@ -1,3 +1,3 @@
 module Elasticity
-  VERSION = '2.8.0'
+  VERSION = '2.9.0'
```

Comment:
Ignore, this is fine. I have another PR to merge in if we are bumping version too. Let me send you